test/invalid/baseline/Int32-keyword.txt

```diff
@@ -0,0 +1,3 @@
+Syntax error at line 1 in Int32-keyword.webidl:
```

Comment:
👍 but could you rename this as `int32array-keyword` because

* I would keep filenames in lowercase, just for consistency
* We may get `int32` in the future, so better be more specific

Sorry to be picky here 😱